catalog/spatial/csw/spatial-csw-source/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/source/reader/GetRecordsMessageBodyReader.java

```diff
@@ -130,8 +151,22 @@ public CswRecordCollection readFrom(Class<CswRecordCollection> type, Type generi
         } finally {
             IOUtils.closeQuietly(inStream);
         }
-
         return cswRecords;
     }
 
+    /* Check HTTP headers for filename */
+    private String handleContentDispositionHeader(String contentDispositionHeader) {
+        if (StringUtils.isNotBlank(contentDispositionHeader)) {
```

Comment:
If you invert this condition you can reduce cyclomatic complexity of this method.